DependencyInjection/Compiler/StaticAsseticHelperPass.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Symfony\Bundle\AsseticBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+class StaticAsseticHelperPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        if (!$container->hasDefinition('assetic.helper.static')) {
+            return;
+        }
+
+        $definition = $container->getDefinition('assetic.helper.static');
+
+        if (!method_exists($definition, 'setShared')) {
+            $definition->setScope('request');
```

Comment:
Ok so if `templating.helper.assets` is scoped on `request`, `assetic.helper.static` should be scoped to. Am I right?